sygnal/utils.py

```diff
@@ -65,7 +71,7 @@ def glob_to_regex(glob):
     return re.compile(r"\A" + res + r"\Z", re.IGNORECASE)
 
 
-def _reject_invalid_json(val):
+def _reject_invalid_json(val: Any) -> None:
```

Comment:
Could also use `NoReturn` here, but I don't think there's any practical benefit to doing so.